app/components/icons/IconArrowLeft.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Svg, { G, Path } from 'react-native-svg';
```

Comment:
check for https://github.com/oblador/react-native-vector-icons

There is no need to implement icons from scratch